notes/NewSiteArchitecture.md

```diff
@@ -0,0 +1,174 @@
+
+The landing page for this section should begin as a more information-rich introduction to GraphQL, explaining why you might use it, and give a brief overview of the constituent parts. Take-aways from this introduction page:
+
+* GraphQL is a query language.
+* GraphQL servers describe a type system, called a "schema".
+* Clients can access a GraphQL server's type system to learn about what's possible.
+* Clients send queries to servers and typically get back JSON.
+* GraphQL servers validate and execute GraphQL queries.
+
+There is then a TOC through the sections and chapters (this is a straw-man list, open to reordering and addition)
+
+* Core Concepts:
+  * Query Language:
+    * Basics (queries, fields, arguments, aliases, comments)
+    * Variables
+    * Fragments
+    * Values
+    * Directives (skip & include)
+  * Type System:
+    * Basics (Schema, Objects & Fields)
+    * Scalars
+    * Lists & NonNull
+    * Interfaces
+    * Unions
+    * Enums
+  * Advanced Topics:
+    * Validation
+    * Execution & Error Handling
+    * Introspection
+    * Mutations
+* Best Practices:
+  * Servers:
+    * Schema Design Guidelines
+    * Paginating Lists
+    * Non-Breaking Changes
+    * Performant Servers (Batching & Caching)
+    * Authentication & Authorization
+    * Serving over HTTP
+  * Clients:
+    * Using Variables
+    * Co-locating Fragments
+    * Caching Results
+    * Persisted Queries
+    * Generating Models
+
+## Code
+
+*Goal:* Introduce open source GraphQL tools along with quick getting started guidelines for each.
+
+*Timeframe:* Top 3 servers described by Sept 12th, remainder by Sept 30th.
+
+This page is all about fulfilling the "Ok I'm sold! Now what?" conundrum. It should first very quickly reintroduce the elements of GraphQL you would expect to see software for as well as offer a quick path towards getting something working.
+
+1) Servers
+
+Explain the purpose of a GraphQL server, that there are servers written for many different languages and environments, and that graphql-js is the reference implementation operated by Facebook.
+
+Each server should contain the following:
+* Logo
+* Name of Project
+* Language/Environment
+* Link to website
+* Getting started (e.g. npm install + code sample)
+
+2) Clients
+
+Explain the purpose of a GraphQL client, that it's okay to just use curl/XHR/fetch, and that clients can offer more value via smart caches and integration with UI frameworks.
+
+Each client should contain similar set of info as servers.
+
+3) Services
+
+Hosted GraphQL-as-a-service have an opportunity to pitch themselves here.
+
+4) Tools
+
+Highlight many common tools and libraries used by GraphQL community, e.g. GraphiQL.
```

Comment:
Are we OK with this turning into a laundry-list of open source tools? I understand we need to feature GraphiQL somewhere because it's critical to know about, but this could quickly turn into a list of 30+ random libraries unless there is a solid criteria for which ones should be included.